content/spin/install.md

```diff
@@ -175,6 +175,7 @@ spin templates install --git https://github.com/fermyon/spin-js-sdk --upgrade
 spin plugins update
 spin plugins install js2wasm --yes
 spin plugins install py2wasm --yes
+spin plugins install cloud --yes
```

Comment:
They don't actually need to do this though, do they?  Spin will automatically install the `cloud` plugin on demand?  (But maybe we prefer ahead-of-time install so they don't get a speed bump when they first log in.)

Reply:
That was my thought process. It feels like it should really be built in rather than a "you're missing something" on the first install

Reply:
Sounds good - thanks!